src/Drawer/Drawer.spec.js

```diff
@@ -48,6 +50,58 @@ describe('<Drawer />', () => {
     assert.strictEqual(paper.hasClass(classes.paper), true, 'should have the paper class');
   });
 
+  describe('enterTransitionDuration property', () => {
+    const enterDuration = 854;
+    const leaveDuration = 2967;
+
+    it('default value should be from standard timings', () => {
+      assert.strictEqual(Drawer.defaultProps.enterTransitionDuration, durations.enteringScreen);
+    });
+
+    it('should be passed to Slide', () => {
+      const wrapper = shallow(<Drawer enterTransitionDuration={enterDuration} />);
+      assert.strictEqual(wrapper.find(Slide).prop('enterTransitionDuration'), enterDuration);
+    });
+
+    it('should be passed to to Modal\'s backdropTransitionDuration when open=true', () => {
+      const wrapper = shallow(<Drawer
```

Comment:
Sorry, was confused by eslint comma-dangle rule there, seemed to be a bit weird.